### ISSUE TITLE
whointel не видит папку

### ISSUE THREAD

**Issue by ghost** [whointel/WhoIntel#12]
Добрый день. 
Программа Whointel не видит папку с чатами игры. хотя она есть и выполняя файл по ссылке из логов открывает
![whointel](https://user-images.githubusercontent.com/108605018/177028132-ac477d8b-eb16-4d4a-bcfb-9a3d3b06894b.png)
![whointel 2](https://user-images.githubusercontent.com/108605018/177028140-6cff0e28-ef40-4c82-bac2-4c40f13c2808.png)
[main.log](https://github.com/whointel/WhoIntel/files/9034181/main.log)
[renderer.log](https://github.com/whointel/WhoIntel/files/9034182/renderer.log)

**Comment by Vizis** [comment]
Yep, i have same problem with current version. there is wrong character after Documents. there is Documents/EVE but should be Documents\EVE
As it is C:\Users\Myuser\Documents/EVE/logs/Chatlogs   but should be C:\Users\Myuser\Documents\EVE\logs\Chatlogs

**Comment by Danno8** [comment]
так же, автор поправь, будь любезен

**Comment by StaroverovAleksey** [comment]
Такая же проблема, буду очень рад, если поправите, сейчас невозможно пользоваться программой

**Comment by ViperDoa** [comment]
Присоеденяюсь к проблеме .... поправьте пожалуста

**Comment by BAXuTOB** [comment]
Как решить данную проблему?

**Comment by ViperDoa** [comment]
Скачал другую прогу там есть лучше Или старую версию ДаннойОтправлено с iPhone2 июля 2023 г., в 15:19, BAXuTOB ***@***.***> написал(а):﻿
Как решить данную проблему?

**Comment by voffkasp** [comment]
можно создать жесткую ссылку по адресу  C:\Users\\%USERNAME%\Documents\EVE\logs\Chatlogs ведущую в C:\Users\\%USERNAME%\OneDrive\Документы\EVE\logs\Chatlogs
делал с помощью NTLinks Maker 
%USERNAME% замените на свое имя пользователя.
